scripts/postinstall.sh

```diff
@@ -5,7 +5,7 @@ set -o pipefail
 patch-package
 echo "✅ All patches applied"
 
-rn-nodeify --install --hack
+rn-nodeify --install --hack 'crypto,buffer,react-native-randombytes,vm,stream,http,https,os,url,net,fs,process'
```

Comment:
can you add a comment above this line that explains why we are doing this, and like what type of things should be in this list?🙏️

Reply:
great comment @brunobar79, 10 out of 10